addon/components/base/bs-contextual-help.js

```diff
@@ -265,7 +278,7 @@ let component = Component.extend(TransitionSupport, {
    * @private
    */
   _renderInPlace: computed('renderInPlace', function() {
-    return this.get('renderInPlace') || typeof document === 'undefined' || !document.getElementById('ember-bootstrap-wormhole');
+    return this.get('renderInPlace') || !this.destinationElement;
```

Comment:
Should we omit a warning if destination element is missing?

Reply:
Makes sense!